include/picongpu/simulation/stage/CurrentInterpolationAndAdditionToEMF.hpp

```diff
@@ -132,7 +140,7 @@ namespace picongpu
                         /* With no current from macroparticles, there is no need for communication.
                          * However we may still have J from the background (if it is activated) in CORE and BORDER.
                          */
-                        if(FieldBackgroundJ::activated)
+                        if(hasCurrentBackground)
```

Comment:
this could become a `if constexpr(hasCurrentBackground)`

Reply:
It could, but I don't think it should. Whether `hasCurrentBackground` is `constexpr` or not is not related to this check - it is not required neither for conditional compilation nor for performance here.